src/services/api.js

```diff
@@ -9,27 +9,30 @@ export async function getTopRatedFilms() {
     let response = await fetch(`${BASE_URL}/movie/popular?api_key=${API_KEY}&language=en-US&page=1`);
     let json = await response.json();
 
-    let propOfFilm = [];
+    const propOfFilm = [];
```

Comment:
@Mmarvinn  pls remove it, it is redundant

Reply:
fixed